Packs/ServiceNow/ReleaseNotes/2_5_10.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### ServiceNow v2
+- Added support for closing incidents in XSOAR using custom close states from Service Now.
```

Comment:
Remove this - this is duplicate to the second point

Reply:
Done.